x-pack/test/cloud_integration/fixtures/saml/saml_provider/kibana.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
All of this `saml_provider` code was stolen from the `x-pack/test/security_api_integration` suite